src/ncp/ncp_base.cpp

```diff
@@ -631,6 +637,53 @@ void NcpBase::HandleRawFrame(const RadioPacket *aFrame)
     return;
 }
 
+// ----------------------------------------------------------------------------
+// MARK: Outbound Radio Packet Handling
+// ----------------------------------------------------------------------------
+
+void NcpBase::HandleRadioTransmit(otInstance *aInstance, RadioPacket *aPacket, bool aFramePending,
+    ThreadError aError)
+{
+    (void)aInstance;
+    sNcpContext->HandleRadioTransmit(aPacket, aFramePending, aError);
+}
+
+void NcpBase::HandleRadioTransmit(const RadioPacket *aPacket, bool aFramePending, ThreadError aError)
+{
+    if (mCurTransmintTID)
+    {
+        SendPropertyUpdate(
+            SPINEL_HEADER_FLAG | SPINEL_HEADER_IID_0 | mCurTransmintTID,
+            SPINEL_CMD_PROP_VALUE_IS,
+            SPINEL_PROP_STREAM_RAW,
+            "ib",
+            aError,
+            aFramePending
```

Comment:
I am treating this as the completion to the set of `SPINEL_PROP_STREAM_RAW` I wasn't exactly sure of the best way to do that. If `PROP_LAST_STATUS` makes more sense to you, I can change it to that.